### PR TITLE
ankisyncd: 2.1.0 -> 2.2.0

### PR DESCRIPTION
**Pull request by matt-snider**
###### Motivation for this change

Currently ankisyncd uses the [tsudoko/anki-sync-server repository](https://github.com/tsudoko/anki-sync-server) which does not appear to maintained. The repository [ankicommunity/anki-sync-server](https://github.com/ankicommunity/anki-sync-server/) is actively maintained. In [this comment](https://github.com/ankicommunity/anki-sync-server/issues/33#issuecomment-711421178) the maintainer goes over the history of the repo and its connection to the original from dsnopek, and the fork from tsudoku.

See #107526

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [x] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by matt-snider** [comment]
I'm evaluating the new version currently to see if there are any issues.

File: pkgs/servers/ankisyncd/default.nix [hunk 1]
```diff
@@ -60,7 +60,7 @@ python3.pkgs.buildPythonApplication rec {
   meta = with lib; {
     description = "Self-hosted Anki sync server";
     maintainers = with maintainers; [ matt-snider ];
-    homepage = "https://github.com/tsudoko/anki-sync-server";
+    homepage = "https://github.com/ankicommunity/anki-sync-server";
     license = licenses.agpl3;
```
Comment (SuperSandro2000) [inline pkgs/servers/ankisyncd/default.nix:64]: agpl3Plus or agpl3Only?

Comment (matt-snider) [reply pkgs/servers/ankisyncd/default.nix:64]: Hey @SuperSandro2000 thanks for taking a look. I didn't realize `licenses.agpl3` was deprecated. I had a look at [the license](https://github.com/ankicommunity/anki-sync-server/blob/develop/COPYING) as well as the licenses of other packages with `agpl3Plus` and `agpl3Only`. It isn't clear to me how I can differentiate between them. Any help would be appreciated!

Comment (SuperSandro2000) [reply pkgs/servers/ankisyncd/default.nix:64]: If they somewhere mention that you can use aGPL3 or at your option any later version it is Plus. I can't find anything like that so it is only.

Comment (matt-snider) [reply pkgs/servers/ankisyncd/default.nix]: Okay, I updated it to agpl3Only - thanks!

**Pull request merged**